.mocharc.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * https://github.com/swc-project/swc/pull/3009
+ *
+ * There are 2 test runners in this repo:
+ * - jest is being used for unit testing node-swc packages with javascript test cases
+ * - mocha is being used for cargo's test requires js runtime to validate its transform.
+ *
+ * This config is for the mocha test runner invoked by cargo to resolve its global setup file.
+ */
+module.exports = {
+  require: require.resolve('./.mocha.setup.js')
```

Comment:
Instead of using CARGO_WORKSPACE_DIR workaround to let cargo resolve path to root pkg, using config file mocha can resolve automatically.